DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part2_lib_func_call/src/lib_rtl_model.cpp

```diff
@@ -4,8 +4,11 @@
 // SPDX-License-Identifier: MIT
 // =============================================================
 #include <sycl/sycl.hpp>
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+using MyInt27 = ac_int<27, false>;
+using MyInt54 = ac_int<54, false>;
```

Comment:
```suggestion
#include "lib_rtl.hpp"

// This emulation model is only used during emulation, so it should functionally
// match the RTL in lib_rtl.v.
```
no need to re-define these types